Fix a dead link (#9729)
kyma-project/kyma: docs/kyma/04-02-local-installation.md

     ```bash
     kyma provision minikube
     ```

>      >**NOTE:** The `provision` command uses the default Minikube VM driver installed for your operating system. For more details, see [this document](https://kubernetes.io/docs/setup/minikube/).

Reviewer: ```suggestion
     >**NOTE:** The `provision` command uses the default Minikube VM driver installed for your operating system. For the list of supported VM drivers, see [this document](https://minikube.sigs.k8s.io/docs/start/).
```
The new link redirects to: https://kubernetes.io/docs/tasks/tools/.
Should we use this link instead? This note means to refer the reader specifically to supported drivers, not more details in general.

Author: I was thinking about adding the link you provided but wasn't sure whether I can use it or not 🤔 Since there's two of us now, I'll go with it, thanks! 😄
